webapp/lib/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "pyroscope",
+  "version": "0.0.1-%VERSION%",
+  "main": "index.lib.js",
+  "browser": "index.js",
+  "license": "MIT"
```

Comment:
Also, we use Apache 2 for license everywhere else, so we should definitely change that.